### DISCUSSION TITLE
My top 3 usability enhancements in v9

### DISCUSSION THREAD

**Discussion by n0rthface** [MarkMpn/Sql4Cds#475] [General]
I just would like to point out my top 3 usability enhancements in v9 which seem small but are really helpful in everyday routine:

- "Copy with headers" in the result grid does not include an empty first column anymore
- Messages window stays at the point where I scroll to instead of moving to the beginning of the messages when refreshing
- "Yes To All" when multiple updates / deletes should be executed (I know it was there in v8 already, but it has to be included in the top 3 list)

@MarkMpn: Thank you very much again for your dedication to maintain and enhance SQL 4 CDS!

**Comment by SiPurdy** [comment]
> @MarkMpn: Thank you very much again for your dedication to maintain and enhance SQL 4 CDS!

Me to on this, thanks @MarkMpn. Sql4Cds has probably saved me thousands of lines of code over the years.
